api/src/main/java/com/codeforcommunity/rest/subrouter/authenticated/ProtectedSchoolRouter.java

```diff
@@ -123,6 +127,16 @@ private void registerCreateReportWithLibrary(Router router) {
     createReport.handler(this::handleCreateReportWithLibrary);
   }
 
+  private void registerCreateReportWithoutLibrary(Router router) {
+    Route createReport = router.post("/:school_id/reports");
```

Comment:
I think we need to separate the three routes for creating reports, rather than registering them all as `POST` `/:school_id/reports`